File: Labs/Lab1/c++/Line.cpp
#include "Line.h"

Line::Line(const std::string &str) : text(str) {}

int Line::getLength() const
{
    return text.length();
}

std::string Line::getText() const
{
    return text;
}

void Line::sortDescending()
{
    std::sort(text.begin(), text.end(), std::greater<char>());
}

File: Labs/Lab1/c#/Line.cs
using System;

namespace LineApp
{
    public class Line
    {
        private string text;

        public Line(string text)
        {
            this.text = text;
        }

        public string getText()
        {
            return text;
        }

        public int getLen()
        {
            return text.Length;
        }

        public string sort()
        {
            string sortedText = new string(text.OrderByDescending(c => c).ToArray());// => convertation

            return sortedText;
        }
    }
}